src/data_preprocessing/counterfactual_data.jl

```diff
@@ -15,6 +19,7 @@ const InputTransformer = Union{
     StatsBase.AbstractDataTransform,
     MultivariateStats.AbstractDimensionalityReduction,
     GenerativeModels.AbstractGenerativeModel,
+    CI.SCM,
```

Comment:
I like to not abbreviate, explicit is good?

Reply:
@mschauer you mean use just:

```julia
import CausalInference
```

instead of 
```
import CausalInference as CI
```

My only concern is on other parts of the code we are going to have lines bigger and I don't know if it is a good practice.